Discovery/Discovery/Presentation/WebDiscovery/URL+PathExtension.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  URL+PathExtension.swift
+//  Discovery
+//
+//  Created by SaeedBashir on 12/18/23.
+//
+
+import Foundation
+
+public extension URL {
+    var appURLHost: String {
+        return host ?? ""
+    }
+    
+    var isValidAppURLScheme: Bool {
+        return scheme ?? "" == URIString.appURLScheme.rawValue ? true : false
```

Comment:
doesn't just `scheme ?? "" == URIString.appURLScheme.rawValue` work here?

Reply:
empty scheme ("") is valid too here (in case URIString.appURLScheme gets empty someway)? 
maybe need to add `&& scheme?.isEmpty == false`

Reply:
> empty scheme ("") is valid too here (in case URIString.appURLScheme gets empty someway)?

It's a defined value, so it shouldn't be needed to address it at the moment.